bitcoin/bitcoind/wallet.go

```
@@ -164,6 +164,11 @@ func (w *BitcoindWallet) CurrentAddress(purpose spvwallet.KeyPurpose) btc.Addres
 	return addr
 }
 
+func (w *BitcoindWallet) NewAddress(purpose spvwallet.KeyPurpose) btc.Address {
+	addr, _ := w.rpcClient.GetNewAddress(Account)
```

Why not propagate this error? Other than that lgtm.